index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Aladdin</title>
+</head>
+
+<body>
+    <div class="container"> 
+    <div class="header">
+        <nav>
+            <ul>
+                <li><h2>Aladdin</h2></li>
+                <li><button class="fas fa-bars hamburger-icon menu"></button></li>
+            </ul>
+        </nav>
+    </div>
+
+    <main>
+        <div class="main">
+            <div class="headline">
+                <h1>Hey There. I am Aladdin</h1>
+                <h3>I am a Software Developer</h3>
+                <p>I can help you build a product, feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, do not hesitate to contact me.</p>
+            </div>
+
```

Comment:
- [ ]  All the `div's` tags should be a `sections ` tags
![fff](https://user-images.githubusercontent.com/61562626/129426040-b7b06405-5ce2-4930-901f-657d991778d7.jpeg)